src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { default as print } from "./print/print";
 export { default as getDefaultPrinter } from "./get-default-printer/get-default-printer";
 export { default as getPrinters } from "./get-printers/get-printers";
+export { isPrintComplete } from "./utils/parse-response";
```

Comment:
could you please change it to `export { default as isPrintComplete }` to be able to import it in a project like this `import { isPrintComplete } from "unix-print"`

Reply:
You can import it as `import { isPrintComplete } from "unix-print"` as it currently is but I'll update it to match the remaining exports